Dockerfile

```diff
@@ -16,11 +16,11 @@ RUN \
 WORKDIR /tmp
 RUN \
  echo "**** getting source code ****" && \
-   wget "https://netix.dl.sourceforge.net/project/llcon/Jamulus/${JAMULUS_VERSION}/Jamulus-${JAMULUS_VERSION}.tar.gz" && \
-   tar xzf Jamulus-${JAMULUS_VERSION}.tar.gz
+   wget "https://github.com/softins/jamulus/archive/r${JAMULUS_VERSION}.tar.gz" && \
```

Comment:
It seems the `softins` is a fork from `corrados`. Can we use official repository?

Reply:
Yes, of course, that was my mistake. PR updated.

Reply:
Sweet, thank you!